### PR TITLE
Integrate with orchestrator and add deploy dev script

### PR DESCRIPTION
**Pull request by skosito**
- added deploy script for development (deploy mock ubt, ubt splitter, approve 100 tokens)
- make events in ubt splitter more uniform for orchestrator usage
- modify readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,22 @@ npx hardhat verify --network ropsten DEPLOYED_CONTRACT_ADDRESS "Hello, Hardhat!"
 # Performance optimizations
 
 For faster runs of your tests and scripts, consider skipping ts-node's type checking by setting the environment variable `TS_NODE_TRANSPILE_ONLY` to `1` in hardhat's environment. For more details see [the documentation](https://hardhat.org/guides/typescript.html#performance-optimizations).
+
```
Comment (skosito) [inline README.md:47]: should we remove this "advanced hardhat" readme (everything above this line)?

File: contracts/Unibright.sol [hunk 2]
```diff
@@ -24,22 +24,12 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  * function.
  */
 contract UBTSplitter is Context, Ownable {
-    event PayeeAdded(
```
Comment (skosito) [inline contracts/Unibright.sol:27]: payeeadded and payeeupdated are same events, it is easier to only throw payeeupdated

File: README.md (hunk 1 above)
Comment (ognjenkurtic) [reply README.md:47]: Yes

File: package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
     "contracts:format-fix": "npx prettier '**/*.{json,sol,md}' --write",
     "format": "chmod u+x format.sh && ./format.sh",
     "contracts:migrate:mainnet": "env-cmd --silent -f .env.dev npx hardhat deploy-contracts --network mainnet",
+    "contracts:migrate:local": "npx hardhat deploy-contracts-dev --network localhost",
```
Comment (ognjenkurtic) [inline package.json:14]: --network hardhat instead of localhost? i know localhost does not work with hardhat config unless it is specifically defined

**Review by ognjenkurtic — APPROVED** [review]

**Pull request merged**